### PR TITLE
Fix/pbs 459

### PR DESCRIPTION
**Pull request by petermisik**
Delete form functionality from FormNavigationView

### CODE REVIEW

**Review by Pete-Hoch — COMMENTED** [review]
This looks great. But the FormNavigationView is going away and being replaced with the EvenOverview. Can you base this off of PR-220 and do this in the new code?

**Comment by petermisik** [comment]
I'll rebase and add the changes.

**Review by Pete-Hoch — APPROVED** [review]
This looks good to me, it's nice to finally be able to delete something.

File: PBLogging/NewNavViews/Form781View/Overview/EventCard.swift [hunk 1]
```diff
@@ -10,20 +10,15 @@ import SwiftUI
 struct EventCard: View {
 
     @ObservedObject var form: Form781
-    @Binding var disableButtons: Bool
+//    @Binding var disableButtons: Bool
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/Overview/EventCard.swift:13]: Just delete.

File: PBLogging/NewNavViews/Form781View/Overview/EventCard.swift [hunk 2]
```diff
@@ -45,7 +41,8 @@ struct EventCard: View {
             // label will not activate the card button.
             HStack {
                 // The EventLabel view will push to a sheet to edit the event data.
-                EventLabel(form: form, disableButtons: $disableButtons)
+//                EventLabel(form: form, disableButtons: $disableButtons)
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/Overview/EventCard.swift:44]: Delete both comments.

File: PBLogging/NewNavViews/Form781View/Overview/EventCard.swift [hunk 3]
```diff
@@ -118,6 +78,7 @@ struct EventCard_Previews: PreviewProvider {
     @State static var displayed = false
 
     static var previews: some View {
-        EventCard(form:form, disableButtons: $displayed)
+//        EventCard(form:form, disableButtons: $displayed)
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/Overview/EventCard.swift:81]: Again, delete comment.

**Review by Pete-Hoch — APPROVED** [review]
This all looks good. Just remove some of the unneeded comments and push it.

**Pull request merged**